test/utils/noderesourcetopologies/noderesourcetopologies.go

```diff
@@ -225,8 +225,11 @@ func accumulateNodeAvailableResources(nrt nrtv1alpha2.NodeResourceTopology, reas
 	klog.Infof("resInfoList available %s: %s", reason, e2enrt.ResourceInfoListToString(resInfoList))
 	return resInfoList, nil
 }
-func SaturateZoneUntilLeft(zone nrtv1alpha2.Zone, requiredRes corev1.ResourceList) (corev1.ResourceList, error) {
+func SaturateZoneUntilLeft(zone nrtv1alpha2.Zone, requiredRes corev1.ResourceList, filter func(*corev1.ResourceList)) (corev1.ResourceList, error) {
```

Comment:
`corev1.ResourceList` is a map so no need to pass it by pointer. This is a language-level inconsistency we can only embrace and we can't fix